.github/workflows/update-whats-new-ids.yml

```diff
@@ -41,64 +40,11 @@ jobs:
           git add ./src/data/whats-new-ids.json
           git diff-index --quiet HEAD ./src/data/whats-new-ids.json || git commit -m 'chore(whats-new-ids): updated ids'
           echo "::set-output name=commit::true"
-
-      - name: Temporarily disable branch protection
-        id: disable-branch-protection
-        uses: actions/github-script@v1
-        with:
-          github-token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
-          previews: luke-cage-preview
-          script: |
-            const result = await github.repos.updateBranchProtection({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              branch: 'develop',
-              required_status_checks: null,
-              restrictions: null,
-              enforce_admins: null,
-              required_pull_request_reviews: null
-            })
-            console.log("Result:", result)
-
-        # Push directly to the `develop` branch so we get the changes included in the release PR
```

Comment:
i think the release branches have some branch protection enabled on them, so we may need this i think?

Reply:
@roadlittledawn ahh didn't realize that, thought they were just branched off develop and named anything but i see the daily/release* in the protection rules! i'll add that in